notary-server-Dockerfile

```diff
@@ -19,4 +19,4 @@ RUN go install \
     ${NOTARYPKG}/cmd/notary-server
 
 ENTRYPOINT [ "notary-server" ]
-CMD [ "-config", "cmd/notary-server/config.json" ]
+CMD [ "-config", "cmd/notary-server/config.json", "-debug"]
```

Comment:
Yes, I guess I've never taken a look inside these Dockerfiles, Diogo added them a while back. It's actually not critical for the moment as these values get overwritten when Ops wrap their own Dockerfile around this, but I've been trying to work out how we can become more self sufficient with deploys which will involve unwrapping the Ops stuff and using the images built by these Dockerfiles directly.